styleguide/_themes/derek/scss/components/_rug.scss

```diff
@@ -1,15 +1,35 @@
-.rug-phone-numbers{
-  margin-top:6px;
+.rug-contact-text {
+  margin-top: 15px;
 }
-.rug-subhead{
-  font-size: 37px;
-  line-height: 24px;
+
+.rug-head {
+  color: $white;
 }
-.rug-contact-text{
-  margin-top: 15px;
+
+.rug-icon {
+  color: $white;
 }
-.rug-contact-link{
-  text-decoration: none;
+
+.rug-link {
+  color: $white;
   padding-left: 5px;
+  text-decoration: none;
+}
+
+.rug-link-hover {
+  &:hover, &:focus {
+    color: $white;
+    text-decoration: underline;
+  }
+}
+
+.rug-phone-numbers {
+  margin-top: 6px;
+  padding: 0;
 }
 
+.rug-subhead {
+  color: $white;
```

Comment:
@bighappyface do we want to define the color within each of these instead of just adding the class to the html or extending .white? Is there a benefit over other options?